trk/src/AppNavigator.js

```diff
@@ -3,39 +3,52 @@ import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
 import LandingScreen from './Screens/Landing';
 import HomeScreen from './Screens/Home';
-import PokemonListScreen from './Screens/ClimbList';
 import ClimbDetail from './Screens/ClimbDetail';
 import ClimbInputData from './Screens/ClimbList';
 
-const HomeStack = createStackNavigator();
-
-function Home(props) {
```

Comment:
Why remove Home, and paste its content in the Stack.Navigator Container?

Reply:
2 points :
1. 2 headers were showing up out of which only one was changing it's heading.
2. both functions were creating similar screen stack so combining into one is better.